CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 --------------------
 
+[3.6.2] 2020-08-12
```

Comment:
might want to change the date here. Otherwise it looks a little time travel-y :p

Reply:
lol just noticed too, ty ty